hack/env/deploy-latest-kubevirt.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+set -e
+
+NAMESPACE=${1:-kubevirt}
+
+oc apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: ${NAMESPACE}
+EOF
+
+# Deploying kuebvirt
+KUBEVIRT_VERSION=v0.36.3
```

Comment:
One more thing. Can you please write here why we choose this version?
Then it will be ready to go.

Reply:
Done @ksimon1